index.js

```diff
@@ -134,4 +138,12 @@ module.exports = (oppiabot) => {
       await runChecks(context, constants.editEvent);
     }
   });
+
+  oppiabot.on('push', async (context) => {

```

Comment:
We want this check only for pull requests not for branches directly pushed any time like develop/release branches. Can we skip for that?